go/vt/vtgate/endtoend/main_test.go

```diff
@@ -152,6 +167,33 @@ create table t2_id4_idx(
 					Type: sqltypes.VarChar,
 				}},
 			},
+			"sequence_test_seq": {
+				Type:   "sequence",
+				Pinned: "80",
+				Columns: []*vschemapb.Column{{
+					Name: "id",
+					// Type: sqltypes.Int64,
+				}, {
+					Name: "next_id",
+					// Type: sqltypes.Int64,
+				}, {
+					Name: "cache",
+					// Type: sqltypes.Int64,
```

Comment:
@deepthi Do we need to list all the columns from a sequence table in VSchema?

Reply:
I don't think so. Something like this should work (in an unsharded keyspace):
```{
    "tables": {
        "test_seq": {
            "type": "sequence"
        }
     }
}

Reply:
Yes, I tested that yesterday it worked. I don't remember the error I was getting before.